rcsb/app/tests-client/testClientUtils.py

```diff
@@ -241,13 +241,54 @@ def testListDir(self):
             logger.exception("Failing with %s", str(e))
             self.fail()
 
+    def testFilePathRemote(self):
+        repoType = self.__repositoryType
+        depId = "D_1000000001"
+        contentType = "model"
+        milestone = None
+        partNumber = 1
+        contentFormat = "pdbx"
+        version = 1
+        filePathResponse = self.__cU.getFilePathRemote(repoType,depId,contentType,milestone,partNumber,contentFormat,version)
+        self.assertTrue(filePathResponse == self.__repositoryFile1, f'error {filePathResponse} {self.__repositoryFile1}')
+
+    def testFilePathLocal(self):
+        if not os.path.exists(self.__repositoryFile1):
+            os.makedirs(os.path.dirname(self.__repositoryFile1), mode=0o757, exist_ok=True)
+            nB = 64
+            with open(self.__repositoryFile1, "wb") as out:
+                out.write(os.urandom(nB))
+        repoType = self.__repositoryType
+        depId = "D_1000000001"
+        contentType = "model"
+        milestone = None
+        partNumber = 1
+        contentFormat = "pdbx"
+        version = 1
+        filename = self.__cU.getFilePathLocal(repoType,depId,contentType,milestone,partNumber,contentFormat,version)
+        print(f'temp file path {filename}')
+        self.assertTrue(os.path.exists(filename), f'error - {filename} does not exist')
+        with open(filename, "rb") as r:
+            bytes = r.read()
+            print(bytes)
```

Comment:
Is this part really necessary for purposes of the test? Might be worth removing if not...